app/assets/javascripts/application.js

```diff
@@ -42,7 +42,4 @@ jQuery(function($) {
       templateDir: templateDir
     });
   }
-
-  var $buttons = $("label.block-label input[type='checkbox']");
-  new GOVUK.SelectionButtons($buttons);
```

Comment:
Why is this being removed?

Reply:
Excellent question. Sorry, I should probably do this in a separate PR (in fact now you've mentioned it I think I will) - I noticed while I was working on this that this code was throwing an error in the console because `GOVUK.SelectionButtons` couldn't be found (turns out it was removed ages ago) so I took it out. I'll put it in a separate PR.